src/officehours_api/serializers.py

```diff
@@ -28,7 +29,23 @@ def get_my_queue(self, obj):
 
         serializer = QueueAttendeeSerializer(meeting.queue, context={'request': self.context['request']})
         return serializer.data
+    
+    def update(self, instance, validated_data):
+        print("***SERIALIZER VALIDATED DATA:", validated_data)
+        profile = validated_data.pop('profile')
```

Comment:
I take it `validated_data` is a dict here? I would typically prefer `validated_data['profile']` (AKA the `__getitem__` accessor) over `.pop(...)` because `pop` mutates the underlying array, which can cause confusion. I do see that `validated_data` is being passed into the `super().update(...)` after the pop, so maybe there's a reason to remove that key that I'm not seeing?

Reply:
My understanding was that the User serializer doesn't need to package the whole profile model with it, it just needs the phone number. So I take the profile out of the validated data, update the user fields with the remaining validated data, then manually update the phone number field.

Reply:
In commit 103543a I changed the `pop()` to just accessing the profile from the validated data, and it seems to work exactly the same.